server/pulp/server/db/manage.py

```diff
@@ -191,6 +193,14 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        # Prompt the user if there are active workers or running tasks
+        if status.get_workers() or filter(lambda x: x.state in constants.CALL_INCOMPLETE_STATES,
+                                          model.TaskStatus.objects(group_id=None)):
+
+            if not _user_input_continue('There are still running workers/tasks,'
```

Comment:
Also what if we don't give the user an option to continue at all. It seems unsafe to continue. If we do continue to prompt we need to reword this to clearly state that it is likely unsafe to continue.